mr_popular/mrpopular.py

```diff
@@ -40,22 +40,22 @@ def connect_to_endpoint(url, headers):
 
 
 def handler(event=None, context=None):
-    bearer_token = os.environ.get("BEARER_TOKEN")
+    bearer_token = boto3.client('ssm').get_parameter(Name='TWITTER_BEARER_TOKEN', WithDecryption=True)['Parameter']['Value']
```

Comment:
Well acktually, shouldn't we set this outside of the handler function so it can reuse the value without making an expensive SSM lookup for every warm start?

Reply:
As it was said on twitter, the ssm client should definitely be outside of the handler.  Where to get the bearer token would depend on how often you think it may change (any executions within ~15 minutes may use the old token if the token is outside the handler).